protected/humhub/modules/file/components/FileManager.php

```diff
@@ -112,10 +113,18 @@ public function findAll()
      */
     public function findStreamFiles($showInStream = true)
     {
-        if ($showInStream) {
-            return $this->find()->andWhere(['show_in_stream' => 1])->all();
+        if ($this->record->isRelationPopulated('files')) {
```

Comment:
Can you please provide a unit test for this?

Reply:
@luke- it's just relation loading, do we really need unit test for this?

Reply:
Hmm it's more about this:

```
            return ArrayHelper::getValue(
                ArrayHelper::index($this->record->files, null, 'show_in_stream'),
                +$showInStream,
                []
            );
```

Reply:
Yeah, seems complex ) but it's just filters show_in_stream = true|false files from all files list. I decided to write this in one line instead of several foreachs. I think we don't need unit for this, it's just filtering, there no any functionality or business logic here. But if you want i'll add test for sure.

Reply:
@gevorgmansuryan Thanks for the explaination. Ok, it's fine as it is.